HISTORY.md

```diff
@@ -1,13 +0,0 @@
-3.0.0:
```

Comment:
usually we try to keep the files, but it looks like it wasn't used in the past either.